2/contrib/jenkins/install-plugins.sh

```diff
@@ -329,6 +330,11 @@ function resolveDependencies() {
 }
 
 function bundledPlugins() {
+    echo "Checking JENKINS_WAR=$JENKINS_WAR"
+    if [ ! -f $JENKINS_WAR ]; then
+        jenkins_version=$(cat $PWD/2/contrib/openshift/jenkins-version.txt )
```

Comment:
add an echo that displays the value of `jenkins_version`

Reply:
done

Reply:
ok now see 

```
2022-02-02T16:54:45.838915482Z Jenkins version: 2.319.2
```

in the image build log.

and https://get.jenkins.io/war-stable/2.319.2/ rendors